Something/WebApplication/wwwroot/js/apps/Global.scss

```diff
@@ -0,0 +1,10 @@
+@import 'AntOverrides.scss';
+
+// todo: to stay until, we find a way to override ant design's variables.
+body,
+h1, h2, h3, h4, h5, h6,
+p, a, label, input, b,
+.ant-btn, .ant-input, .ant-form-item,
+.ant-alert {
+    font-family: 'Lato', 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif !important;
+}
```

Comment:
BEMifiy this

Reply:
All these are temporary font overrides only. Need to setup https://github.com/ant-design/ant-design/blob/master/components/style/themes/default.less after which these style will be removed.